assets/js/blocks/product-gallery/inner-blocks/product-gallery-thumbnails/edit.tsx

```diff
@@ -49,7 +52,7 @@ export const Edit = ( {
 		<>
```

Comment:
Not actually related to this PR, but I think we can remove the `<>` (React Fragment) from here, because we have only one child component that is also a `div`

Reply:
Good point. Thank you for the review 🙏